### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception  java.lang.IllegalArgumentException: Argument for @NotNull parameter 'virtualFile' of com/microsoft/azure/toolkit/intellij/common/AzureArtifact.createFromFile must not be null

### ISSUE THREAD

**Issue by silvioricardo87** [microsoft/azure-tools-for-java#6271]
IntelliJ build version: 2021.3 IU-213.5744.223
OS: Mac OS X
JDK: JetBrains s.r.o. 11.0.13
Plugin version: 3.59.0-2021.3
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,0,0,1440x900,layout=javax.swing.JRootPane$RootLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=null, throwable=java.lang.IllegalArgumentException: Argument for @NotNull parameter 'virtualFile' of com/microsoft/azure/toolkit/intellij/common/AzureArtifact.createFromFile must not be null
	at com.microsoft.azure.toolkit.intellij.common.AzureArtifact.$$$reportNull$$$0(AzureArtifact.java)
	at com.microsoft.azure.toolkit.intellij.common.AzureArtifact.createFromFile(AzureArtifact.java)
	at com.microsoft.azure.toolkit.intellij.common.AzureArtifact.createFromFile(AzureArtifact.java:36)
	at com.microsoft.azure.toolkit.intellij.common.AzureArtifactManager.getAzureArtifactById(AzureArtifactManager.java:98)
	at com.microsoft.azure.toolkit.intellij.webapp.runner.webappconfig.slimui.WebAppDeployConfigurationPanel.lambda$setValue$13(WebAppDeployConfigurationPanel.java:233)

**Comment by wangmingliang-ms** [comment]
this issue should have been fixed by commit 65caf143, can you try with the latest versions

**Comment by wangmingliang-ms** [comment]
close as it has been fixed since v3.69.0, you can upgrade to the latest and try. Feel free to reopen it when you encounter this error again.
